src/modules/allowlist/token.ts

```diff
@@ -0,0 +1,53 @@
+import configs from '@/config';
+import { getRpcUrl } from '@/modules/network';
+import { JsonRpcProvider } from '@ethersproject/providers';
+import { Contract } from '@ethersproject/contracts';
+import { callGitHubWebhook } from '@/modules/github';
+import { ALLOWLIST_TOKEN_ENDPOINT } from '@/constants';
+
+export async function allowlistTokens(chainId: number, tokenAddresses: string[]) {
+  for (const address of tokenAddresses) {
+    await allowlistToken(chainId, address);
```

Comment:
isn't it better to put in .map and Promise.all

Reply:
True. Fixed.